src/SDKs/Automation/Management.Automation/Generated/IAutomationClient.cs

```diff
@@ -256,9 +267,9 @@ public partial interface IAutomationClient : System.IDisposable
         IDscNodeConfigurationOperations DscNodeConfiguration { get; }
 
         /// <summary>
-        /// Gets the IWatcherOperations.
+        /// Gets the INodeCountInformationOperations.
         /// </summary>
-        IWatcherOperations Watcher { get; }
+        INodeCountInformationOperations NodeCountInformation { get; }
```

Comment:
This is a breaking change. This will need a minor version bump in the csproj (`3.2.0-preview`)